nextcord/ext/commands/bot.py

```diff
@@ -707,6 +733,11 @@ def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
             Defaults to ``None``.
 
             .. versionadded:: 1.7
+        extras: Optional[:class:`dict`]
+            A mapping of kwargs to values to be passed to your
+            cog's ``__init__`` method as key word arguments.
+
+            .. versionadded:: 2.0.0a4
```

Comment:
a4 should not be added as thats ``alpha`` and ``serial 4``

Reply:
Yep. What should it be then? Just 2.0.0